### DISCUSSION TITLE
Copilot can not use in JetBrains IDEA

### DISCUSSION THREAD

**Discussion by darkings1990** [community/community#118384] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I have log in my account darkings1990, but copilot still not work

<img width="776" alt="image" src="https://github.com/community/community/assets/71364304/60eb1344-1378-416d-8d5d-512f1190c0cb">

<img width="1151" alt="image" src="https://github.com/community/community/assets/71364304/7c57cf45-6b81-4e9d-82f6-62efe6938a82">

![image](https://github.com/community/community/assets/71364304/9e0aaa43-344c-4864-9571-6f6e2e95b844)

**Comment by AdityaRoy999** [comment]
**1.)** Try to _Reinstall Copilot Extension_  and _Re-login Your account_ , Sometimes This may Fix The issues you are having  or maybe try a different IDE 
like VSCode instead of the one  you are currently using.
**2.)** _Verify with your organization_ if the Co-pilot has been set up properly , sometime improper setting can lead to the particular member of the organization not been able to access it.
**3.)** Check if the account you are using is linked and verified by your organization.
If you have further Questions Please feel free to ask.

**Comment by ayim** [comment, marked as answer]
@darkings1990 in this scenario, there is nothing we can do beyond you using the Contact Support button to reach out to find out more about the copilot token issue.
